### DISCUSSION TITLE
Welcome to hacktoberfest2021 Discussions!

### DISCUSSION THREAD

**Discussion by keshavsingh3197** [keshavsingh4522/hacktoberfest2021#659] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by s-a-c-h-i-n** [comment]
Hi,
The PR made here is showing as Excluded Project in the Hacktoberfest profile. Do you know why it is showing that?

**Reply by keshavsingh3197** [reply]
@s-a-c-h-i-n 
I don't know it is excluded or not but
I have recommended in readme.md file that don't commit or pull in this repo. I have mentioned other repo in readme.md file or or at the bottom of this repo.
